koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -527,8 +480,7 @@ def upload_build_annotations(self, build_response):
             incremental_upload(self.session, ANNOTATIONS_FILENAME, f, self.getUploadPath(),
                                logger=self.logger)
 
-    def handle_build_response(self, build_response, arch=None):
```

Comment:
Should we have kept the BuildResponse class? It would have made the koji-c changes smaller

Another thing is that each `self.osbs().some_method(build_id)` call does a new HTTP request for the PipelineRun data, where before the BuildResponse methods didn't do any additional HTTP requests

Reply:
they didn't call new HTTP request for those annotations/labels etc, but those are one time requests

we aren't doing that repeatedly

we start pipelinerun, then read logs, and then do few requests to fetch those data

but now we have at least all logic related to pipeline run only in one place

Reply:
Ok, should be fine